### PR TITLE
run tests in travis

### PR DESCRIPTION
**Pull request by ankilosaurus**

### CODE REVIEW

**Comment by grosser** [comment]
please use rebase instead of merging to keep commits clean

File: pkg/remediator/crashloopbackoff_rescheduler_test.go [hunk 1]
```diff
@@ -119,7 +119,7 @@ func (suite *TestCrashLoopBackOffReschedulerSuite) SetupTest() {
 }
 
 func (suite *TestCrashLoopBackOffReschedulerSuite) testRemediator() {
-	ctx, cancel := context.WithCancel(suite.ctx)
```
Comment (grosser) [inline pkg/remediator/crashloopbackoff_rescheduler_test.go:122]: can delete the suite.ctx setter above then ?

**Comment by ankilosaurus** [comment]
> please use rebase instead of merging to keep commits clean

yup realized closing this. opened new one.

**Comment by grosser** [comment]
rebase + push --force works too and produces less PRs ;)

**Pull request closed without merging**